theano/sandbox/gpuarray/type.py

```diff
@@ -751,3 +758,34 @@ def c_code_cache_version(self):
 declaration of an operation.
 """
 gpu_context_type = GpuContextType()
+
+
+# THIS WORKS But CudaNdarray instances don't compare equal to one
```

Comment:
Maybe update the comment to say "GpuArray", and mention that this is not a new issue as it is already present in CudaNdarray?